homework/add-even/addEven.hpp

```diff
@@ -2,7 +2,11 @@
 #include <vector>
 
 int addEven(const std::vector<int>& numbers) {
-    // TODO: Your implementation goes here
-    // Below return is only to make this function compile now
-    return -1;
+    int result = 0;
+    for(auto i : numbers){
+        if(i%2 == 0){
```

Comment:
Add space after `if`, `for`, `while` `function` etc...

Add space between operator like `+` `-` `/` `:` `&&` etc..